src/smart-components/role/add-role-new/review-template.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, TextContent, Title } from '@patternfly/react-core';
+import './review.scss';
+
+const ReviewTemplate = ({ formFields }) => {
+
+    const review = formFields[0][0];
+    return <React.Fragment>
+        <Title headingLevel="h1" size="xl" className='ins-c-rbac__gutter-sm'>
+            Review details
+        </Title>
+        <TextContent className='ins-c-rbac__gutter-md'>
+            <Text>
+                Review and confirm the details for your role, or click Back to revise.
+            </Text>
+        </TextContent>
+        {[ [{ ...review }] ]}
```

Comment:
Actually, no need to assign to a variable, you can just use the value directly
```suggestion
        {[ [{ ...formFields?.[0]?.[0] }] ]}
```